pkgs/development/compilers/llvm/18/default.nix

```diff
@@ -146,6 +146,11 @@ in let
         # https://reviews.llvm.org/D51899
         ./clang/gnu-install-dirs.patch
         ../common/clang/add-nostdlibinc-flag.patch
+        (fetchpatch {
+          url = "https://github.com/llvm/llvm-project/commit/5b09e6bf5aaeb21256e86455f8d6c42d01f71b95.patch";
```

Comment:
Shouldn't this be https://github.com/llvm/llvm-project/commit/48c1364200b5649dda2f9ccbe382b0bd908b99de, the commit that actually exists in that repo?

Would be nice for the patch to have a name too, so it's obvious what it's for.

Reply:
this is done